R/as-tibble.R

```diff
@@ -1,26 +1,77 @@
+#' Flatten a Crunch Cube
+#'
+#' Crunch Cubes can be expressed as a long data frame instead of a
+#' multidimensional array. In this form each dimension of the cube is a variable
+#' and the cube values are expressed columns for each measure. This is useful
```

Comment:
"and the cube values are expressed columns for each measure." ?> "and the cube values are expressed as columns for each measure."